x-pack/plugins/security_solution/common/detection_engine/build_exceptions_query.ts

```diff
@@ -196,44 +156,48 @@ export const buildExceptionItemEntries = ({
   exclude: boolean;
 }): string => {
   const and = getLanguageBooleanOperator({ language, value: 'and' });
+  const not = getLanguageBooleanOperator({ language, value: 'not' });
   const exceptionItem = lists
     .filter(({ type }) => type !== 'list')
     .reduce<string[]>((accum, listItem) => {
-      const exceptionSegment = evaluateValues({ item: listItem, language, exclude });
+      const exceptionSegment = evaluateValues({ item: listItem, language });
       return [...accum, exceptionSegment];
     }, []);
-
-  return exceptionItem.join(` ${and} `);
+  const exceptionQuery = exceptionItem.join(` ${and} `);
```

Comment:
Here we want to join with OR, right? Since now up [here](https://github.com/elastic/kibana/pull/71512/files#diff-aa45a3a07dc0eb639b4a4ce2458a668aR59) the `excluded` operator gets `not` in front of the expression?

I think we can also remove the `exclude` parameter entirely and allow callers of the function to surround the expression with an additional `not` if they want to.